tools/authors/manage.php

```diff
@@ -183,12 +183,12 @@ function deleteAuthor(check, author) {
             check.checked = false;
             return;
         }
-        eval("form.new_enabled_author_"+author+".disabled=true;");
+        document.getElementsByName("new_enabled_author_" + author)[0].disabled = true;
```

Comment:
These `document.getElementsByName()[0]` are safe in this context because the names are unique in the page.